dev-protocol.conf

```diff
@@ -0,0 +1,6 @@
+canton.participants.sandbox.parameters.alpha-version-support = true
```

Comment:
I presume you needed this file for running the sandbox locally? If so, since we don't need to do that anymore then we should remove this file.

Reply:
Better to remove it - if we leave it on `main` then we'd need to support it and we'd leave ourselves open to questions on what this is (it's already part of our docs on how to run the sandbox).